doc/README.hbs

```diff
@@ -17,6 +17,53 @@ Supports:
 - GNU/Linux distributions that include [util-linux](https://github.com/karelzak/util-linux) and [udev](https://wiki.archlinux.org/index.php/udev).
 - Mac OS X.
 
+The `drivelist` core consists of a set of scripts built with technologies that
```

Comment:
Maybe worth adding an inline comment at the top of this file, telling people to edit `doc/README.hbs` rather than `README.md` directly? ;)